project-base/storefront/styles/globals.css

```diff
@@ -190,7 +200,7 @@ input[type='number'] {
 
 .Toastify__toast-theme--colored.Toastify__toast--success
     .Toastify__progress-bar-theme--colored.Toastify__progress-bar--success {
-    @apply !bg-greenLight;
+    @apply !bg-[#81f7da];
```

Comment:
I would rather define some green color and use it there

Reply:
This is not part of the scope which should be solved here, because there is no design for Toasts yet. But I solved it temporarily by some adjustments. Let's wait with other changes for new design then we can adjust.